hot-reload.md

```diff
@@ -281,3 +284,11 @@ to:
 
 In these situations, hot reload generates a diagnostic message and fails without
 committing any changes.
+
+## How it works
+
+When hot reload is invoked, the host machine looks at the edited code since the last compilation. Each library with changed code, the application's main library, and the libraries from the main library leading to affected libraries are recompiled. In Dart 2, those libraries' Dart source code are turned into [kernel files](https://github.com/dart-lang/sdk/tree/master/pkg/kernel) and sent to the mobile device's Dart VM.
```

Comment:
.."the host machine" -> Do you mean framework?

I think it would be clearer to break down the second sentence:

last compilation, and the following libraries are recompiled:

* Any libraries with changed code.
* The app's main library.
* The libraries from the main library leading to affected libraries.

Reply:
Ah, it's clearer this way. Thanks. 

The host machine is the machine with the source code running the flutter tool (as opposed to the mobile device).